### PR TITLE
GRIFFON ENGINE: Fix final boss is sluggish

### PR DESCRIPTION
**Pull request by harshbawari**
This PR fixes the Final Boss is Sluggish issue.

Game can now be completed by killing the final boss.

### CODE REVIEW

**Comment by bluegr** [comment]
The commits should be like:
GRIFFON: Blah blah

Please, fix them. Also, part of this has already been submitted in your previous poll request

File: graphics/transparent_surface.cpp [hunk 1]
```diff
@@ -749,43 +749,30 @@ void TransparentSurface::setAlphaMode(AlphaType mode) {
...
 distribution.
-
 Andreas Schiffler -- aschiffler at ferzkopp dot net
 =============================================================================
-
-
 The functions have been adapted for different structures and coordinate
 systems.
-
 */
```
Comment (bluegr) [inline graphics/transparent_surface.cpp]: These whitespace changes should not be submitted

File: engines/griffon/draw.cpp [hunk 2]
```diff
@@ -183,6 +183,8 @@ void GriffonEngine::drawHud() {
 	//sprintf(line, "_fps: %i, map: %i, exp: %i/%i", (int)_fps, _curmap, _player.exp, _player.nextlevel);
 	//drawString(_videobuffer, line, 0, 0, 0);
 
+	_itemyloc = 0;
+
```
Comment (bluegr) [inline engines/griffon/draw.cpp]: This change is new compared to the ones done in pull request #1996

File: engines/griffon/resources.cpp [hunk 3]
```diff
@@ -625,6 +631,8 @@ void GriffonEngine::loadMap(int mapnum) {
 			_npcInfo[i].spellDamage = 60;
 
 			_npcInfo[i].walkspd = 1;
+			_npcInfo[i].frame2 = 0;
+			_npcInfo[i].attackattempt = 0;
```
Comment (bluegr) [inline engines/griffon/resources.cpp]: This change is new compared to the ones done in pull request #1996

**Comment by harshbawari** [comment]
Added GRIFFON ENGINE to commit message

**Comment by sev-** [comment]
@harshbawari please follow what @bluegr mention and fix the commit log messages.

You may read about our commit guidelines on our Wiki on the Developer Central page.

**Comment by sev-** [comment]
Also, please rebase you tree to avoid the duplicates.

**Comment by sev-** [comment]
Thanks for fixing the commit log messages. What about other things which we discussed: rebase it and remove incorrect changes outside of griffon engine?

**Comment by harshbawari** [comment]
I did rebase it. For those whitespaces, I saw bluegr reverted that merge.

**Comment by sev-** [comment]
If you rebased it, then why it is based on the code 7 days ago? Look here: https://github.com/harshbawari/scummvm/commits/final-boss-sluggish

**Comment by sev-** [comment]
Also, although with these changes actor started to move, still it does not move smoothly as on previous screen. As such, the commits do not really fix the issue fully.

**Comment by sev-** [comment]
> I did rebase it. For those whitespaces, I saw bluegr reverted that merge.

Yes, but you are reintroducing this change.

**Pull request closed without merging**